src/utils.rs

```diff
@@ -41,20 +41,20 @@ pub(crate) fn print_version() {
     println!("version: {version}");
 }
 
-pub(crate) fn print_run_executable_command() {
-    let executable_name = format!(
-        "subspace-cli-{}-{}-v{}-alpha",
-        env::consts::OS,
-        env::consts::ARCH,
-        env!("CARGO_PKG_VERSION")
-    );
-
-    #[cfg(target_os = "windows")]
-    let executable_name = format!("{executable_name}.exe");
-
-    let command = format!("`./{executable_name} farm`");
+pub(crate) fn generate_run_executable_command() -> Result<String> {
+    let exec_full_path = std::env::current_exe()?;
+    let exec_dir = exec_full_path.parent().expect("directory cannot be empty");
+    let current_dir = std::env::current_dir()?;
+
+    // if current dir is equal to the dir of the exec, just recommend the filename
+    // else, recommend the full path
+    let exec_name = if current_dir.eq(exec_dir) {
+        format!("./{:?}", exec_full_path.file_name().expect("filename cannot be empty"))
+    } else {
+        format!("{exec_full_path:?}")
+    };
 
-    println!("{command}");
+    Ok(format!("`{exec_name} farm`"))
```

Comment:
I think you can just use `std::env::current_exe()` instead of this function

Reply:
If the user runs the executable from another directory, the suggested command wont work via copy paste. This should be better for users, and wasnt a big hassle to me. I think it is useful

Reply:
This way, it will always work via simple copy paste without any change required from the end-user

Reply:
While in the same directory with executable, i didnt want to print a huge line a command including all the path. And printing only the name of the executable wasnt cutting it when the directory was not the same

Reply:
Okay, how about:
```rust
PathBuf::from(std::env::args().next().expect("Command always exists"))
```